README.md

```diff
@@ -127,8 +125,18 @@ To build Ceph, follow this procedure:
 
        ninja install
 
-    
+## Build Types
+
+``do_cmake.sh`` by default creates a "debug build" of Ceph (assuming .git exists).
+A ``Debug`` build which can be up to five times slower than a non-debug build.
```

Comment:
We can measure speed but not slowness.

Suggest:

\`\`do_cmake.sh\`\` by default creates a debug build, assuming that \`\`.git\`\` exists. A  debug build takes as much as five times longer to complete than a non-debug build.

Reply:
It's not about time to complete the build but actual performance of the cluster itself at runtime.

Reply:
Ah.  Suggest then "The runtime performance of a debug build may be as little as 20% of that of a regular build".